### PR TITLE
Fetch data - 

### PR DESCRIPTION
**Pull request by IsaackMwenda**
Hi @ritazagoni have added the initial scripts to create UUID table and fetch raw data .PTAL

### CODE REVIEW

File: run_scripts/2_fetch_raw_data.sh [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    "icraf_s01_e02_activation"
+    "icraf_s01_e03_activation"
+    "icraf_s01_e04_activation"
+    "icraf_s01_e05_activation"
+    "icraf_s01_e06_activation"
+)
+
+SURVEYS=(
+    "icraf_s01_demogs"
+    "icraf_s01_fus"
```
Comment (as2388) [inline run_scripts/2_fetch_raw_data.sh]: what is fus?

File: run_scripts/2_fetch_raw_data.sh [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+)
+
+./run_scripts/checkout_rapid_pro_tools.sh "$RP_DIR"
+
+mkdir -p "$DATA_ROOT/Raw Data"
+
+# Export radio show messages_datasets
+cd "$RP_DIR/fetch_runs"
+for SHOW in ${SHOWS[@]}
+do
```
Comment (as2388) [inline run_scripts/2_fetch_raw_data.sh]: I suggest you build this project using the current master branch of ADSS, as this is the project where we're currently pushing how the pipelines work and what they can do.

In particular here, I re-wrote this to a be a Python script on ADSS (which reads login credentials from buckets and configuration files. It's still making subprocess calls for now, but will soon be calling a new Rapid Pro library directly).

File: run_scripts/checkout_rapid_pro_tools.sh [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    echo "Usage: ./checkout_rapid_pro_tools.sh <rapid-pro-tools-dir>"
+    echo "Ensures that a copy of the RapidProTools project exists in  'rapid-pro-tools-dir' by cloning/fetching as "
+    echo "necessary, and checking-out the version needed by this project"
+    exit
+fi
+
+RAPID_PRO_TOOLS_DIR="$1"
+
+RAPID_PRO_TOOLS_REPO="https://github.com/AfricasVoices/RapidProTools.git"
+TAG="v0.2.4"
```
Comment (as2388) [inline run_scripts/checkout_rapid_pro_tools.sh]: I would use the (recently released) v0.2.5 (https://github.com/AfricasVoices/RapidProTools/releases/tag/v0.2.5).

Very little has changed, but you may as well enjoy the benefit of better error messages.

**Review by as2388 — CHANGES_REQUESTED** [review]

File: run_scripts/2_fetch_raw_data.sh (hunk 1 above)
Comment (IsaackMwenda) [reply run_scripts/2_fetch_raw_data.sh]: Follow up survey

File: run_scripts/2_fetch_raw_data.sh (hunk 2 above)
Comment (IsaackMwenda) [reply run_scripts/2_fetch_raw_data.sh]: Thank you, let me take a look at it.

File: README.md [hunk 4]
```diff
@@ -1 +1,58 @@
...
+```
+
+where `data-root` is an absolute path to the directory in which all pipeline data should be stored. 
+The UUID table will be saved to a file in the directory `<data-root>/UUIDs`.
+
+### 2. Fetch Raw Data
+Next, fetch all the raw data required by the pipeline from Rapid Pro by running the following command in 
+the `run_scripts` directory:
+
+`pipenv run python 2_fetch_raw_data.py <user> <avf-credentials-service-account-file-path> <pipeline-config-file-path> <rapid-pro-root> <data-root>`.
```
Comment (as2388) [inline README.md:42]: Thanks for this - I will be copying this back to ADSS :)

**Review by as2388 — APPROVED** [review]

File: run_scripts/2_fetch_raw_data.sh (hunk 1 above)
Comment (as2388) [reply run_scripts/2_fetch_raw_data.sh]: Is it possible to rename the flow to icraf_s01_follow_up_survey?

**Pull request merged**